packages/launchpad/src/launchpad.tsx

```diff
@@ -1,15 +1,19 @@
 import { useLaunchpad } from '@automattic/data-stores';
 import Checklist from './checklist';
+import { Task } from './types';
```

Comment:
This looks like it should be a type import.
```suggestion
import type { Task } from './types';
```